src/components/OSMElementTags/OSMElementTags.jsx

```diff
@@ -39,6 +39,11 @@ const OSMElementTags = props => {
   }
 
   const [selectedFeatureId, setSelectedFeatureId] = useState(featureIds[0])
+  
+  useEffect(() => {
+    setSelectedFeatureId(featureIds[0])
+  }, [featureIds])
+
   const widgetLayoutProps = { featureIds, selectedFeatureId, setSelectedFeatureId }
 
   const { isLoading, isError, error: fetchErr, data: element } = useQuery({
```

Comment:
I also wanted to add `refetchOnWindowFocus: false,` here.

Most of the time the users will re-focus the window after doing their modification so they will "discard" the updated data because they will switch to another task. I think it make sense to avoid this behaviour here.

Also the OSM History widget does not have this behaviour, hence it make sense to have the same behaviour between both.